musicbrainz-server/templates/default/sv-musicbrainz-server-run.erb

```diff
@@ -5,10 +5,14 @@ set -e
 cd mb_server
 umask 022
 exec \
-    env MUSICBRAINZ_USE_PROXY=1 starman -Ilib \
-        --preload-app \
-        -E deployment \
-        --listen musicbrainz-server.socket \
-        --workers <%= @variables[:nproc] || 5 %> \
-        --user www-data \
-        --group www-data
+    env MUSICBRAINZ_USE_PROXY=1 \
+        setuidgid www-data \
+            start_server --path musicbrainz-server.socket -- \
```

Comment:
Is it sufficient to have this only here, or shouldn't the path for listening still need to be put into the plackup command? I though it only went in start_server so it could preallocate it, but that whatever ran under it still needed to be passed the info, but perhaps that's what being compatible with Server::Starter means, knowing how to get this info from here?